### PR TITLE
[frio] Get PHP Configuration for upload limit and display in /admin summary site

### PR DESCRIPTION
**Pull request by JonnyTischbein**
According to #5547 I get the PHP configuration values for upload limits and display them in the /admin summary site.

I still do not know how to adjust these values reliable (see https://github.com/friendica/friendica/issues/5547#issuecomment-425041824).

Does the place where I put the table fits?

### CODE REVIEW

File: mod/admin.php [hunk 1]
```diff
@@ -908,6 +908,7 @@ function admin_page_summary(App $a)
 
 	$queues = ['label' => L10n::t('Message queues'), 'queue' => $queue, 'workerq' => $workerqueue];
 
+	$php_settings = [L10n::t('PHP Values'), ['upload_max_filesize' => ini_get('upload_max_filesize'), 'post_max_size' => ini_get('post_max_size'), 'memory_limit' => ini_get('memory_limit')]];
```
Comment (MrPetovan) [inline mod/admin.php]: Please use an associative array withe the `title` and `values` keys. Make sure to update the references to `$php` in the templates.

File: mod/admin.php [hunk 2]
```diff
@@ -909,6 +909,15 @@ function admin_page_summary(App $a)
 	$queues = ['label' => L10n::t('Message queues'), 'queue' => $queue, 'workerq' => $workerqueue];
 
 
+	$r = q("SHOW variables LIKE 'max_allowed_packet'");
```
Comment (JonnyTischbein) [inline mod/admin.php]: I didn't know how else to do this query. DBA::selectFirst would do a SELECT

**Comment by JonnyTischbein** [comment]
> You also need to add the PHP values table to the base view/templates/admin/summary.tpl for non-frio themes as well.

Can I use bootstrap CSS in non-frio themes ?

File: mod/admin.php (hunk 2 above)
Comment (MrPetovan) [reply mod/admin.php]: You did good, the DBA::select* methods have a very specific usage and it doesn't cover server variables.

**Comment by MrPetovan** [comment]
It looks good, do you have a couple screenshots from frio/vier to show?

**Comment by JonnyTischbein** [comment]
frio
frio

vier
vier

**Comment by MrPetovan** [comment]
Great, thank you for your work!

**Pull request merged**